gui/mountPrimaryWindow.py

```diff
@@ -42,6 +43,17 @@
 
 bucketOptions = ['s3storage', 'azstorage']
 mountTargetComponent = 3
+cloudfuseCli = 'cloudfuse'
+mountDirSuffix = ''
+if platform == 'win32':
+    # on Windows, the cli command ends in '.exe'
+    cloudfuseCli += '.exe'
+    # on Windows, the mound directory must not exist before mounting,
+    # so name a non-existent subdirectory of the user-chosen path
+    mountDirSuffix = 'cloudFuse'
```

Comment:
I think we should eventually change the mount directory to just cloudfuse for consistency.

Reply:
Done.